models/product-type/src/product-type/product-type-draft/presets/sample-data-goodstore/product-sets.ts

```diff
@@ -0,0 +1,32 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import {
+  AttributeDefinitionDraft,
+  AttributeLocalizableTextType,
+  AttributeSetTypeDraft,
+} from '../../../../index';
+import type { TProductTypeDraftBuilder } from '../../../types';
+import * as ProductTypeDraft from '../../index';
+
+const productSets = (): TProductTypeDraftBuilder =>
+  ProductTypeDraft.presets
+    .empty()
+    .key('product-sets')
+    .name('Product sets')
+    .description('products also sold as sets with their variants')
+    .attributes([
+      AttributeDefinitionDraft.presets
+        .empty()
+        .name('type')
+        .type(
+          AttributeSetTypeDraft.random().elementType(
+            AttributeLocalizableTextType.random()
+          )
+        )
+        .label(LocalizedString.presets.empty()['en-GB']('Type')['de-DE']('Typ'))
```

Comment:
In general, if US is missing in the localization data, do we include it?

Reply:
Yes!